lib/testing-helpcenter/data/help_center_data.rb

```diff
@@ -31,5 +31,16 @@ def self.main_page_block_links
     def self.footer_links
       @footer_links ||= YAML.load_file("#{__dir__}/footer_links_info.yml")
     end
+
+    def self.main_page_faq_info
+      {
+        'What license is ONLYOFFICE distributed under?': ['ONLYOFFICE Legal Notices and Terms of Use'],
```

Comment:
I think every data info more compilcated than 3-4 lines should be placed in `yml` file